src/Bounds.cpp

```diff
@@ -645,14 +639,20 @@ class Bounds : public IRVisitor {
             min = Expr(); max = Expr(); return;
         }
 
+        bool const_condition = !expr_uses_vars(op->condition, scope);
```

Comment:
Can't construct a select node where one of the paths is an undefined Expr, so we just return an undefined Expr.

I added a comment explaining the div case (after thinking about it for while).